library/psa_crypto.c

```diff
@@ -4484,7 +4484,7 @@ static psa_status_t psa_key_derivation_input_raw(
     if( PSA_ALG_IS_TLS12_PRF( kdf_alg ) ||
              PSA_ALG_IS_TLS12_PSK_TO_MS( kdf_alg ) )
     {
-        // TODO
+        // To do: implement this
```

Comment:
"avoid the magic word that our CI rejects"
What value do we get from checking for "TODO" in our code? Should we also check for "To do" and "FIXME"?

Reply:
No idea, and no.

Reply:
I've raised https://github.com/ARMmbed/mbedtls/issues/2587 so we can address this later.

I suspect the motivation was to enable us to see TODO issues more clearly by removing them from the code and putting them into GitHub instead. I don't see why we couldn't do both, though.